invenio_rdm_records/resources/resources.py

```diff
@@ -76,10 +76,89 @@ def p(route):
                 p(routes["access-request-settings"]),
                 self.update_access_settings,
             ),
+            route("POST", p(routes["delete-record"]), self.delete_record),
+            route("POST", p(routes["restore-record"]), self.restore_record),
+            route("GET", p(routes["list-all"]), self.search_all),
+            route("POST", p(routes["set-record-quota"]), self.set_record_quota),
+            # TODO: move to users?
```

Comment:
What does this mean? Let's don't add TODOs that will never be addressed :)
Otherwise, please move it under user if possible.

Reply:
I am not sure why it was added to record service in the first place ping @zzacharo . I evaluated moving it to users-resources but users-resources module normally would not know about RDMUserQuota model so if I start moving it now, it would trigger an avalanche of changes, therefore if we decide to do that now, we should be sure where to place it

Reply:
I placed it in rdm-records as it was the fastest way to integrate the feature and I didn't want to go all the evaluation of moving it in users-resources (which I am not sure it even belongs to)... we can remove the TODO, I added more of a note in case someone was confused finding that code there at the moment.